vendor/bat-native-ads/src/bat/ads/internal/ads/search_result_ad_test.cc

```diff
@@ -33,59 +40,107 @@ class BatAdsSearchResultAdIntegrationTest : public UnitTestBase {
   }
 };
 
-TEST_F(BatAdsSearchResultAdIntegrationTest, TriggerViewedEvent) {
+TEST_F(BatAdsSearchResultAdIntegrationTest, TriggerViewedEvents) {
   // Arrange
+  const int unblinded_tokens_count =
+      UnblindedTokensPermissionRule::GetUnblindedTokensMinimumThreshold() + 1;
+  ForceUnblindedTokensPermissionRule(unblinded_tokens_count);
```

Comment:
How come we cannot just call `ForcePermissionRules()`?

Reply:
We are firing two ad events here, but `ForcePermissionRules()` allows us to fire only one ad event.

Reply:
Added `privacy::SetUnblindedTokens(20);` call